ext/gmp/tests/gh9308.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Bug GH-9308: GMP throws the wrong error when a GMP object is passed to gmp_init()
+--SKIPIF--
+<?php if (!extension_loaded("gmp")) print "skip"; ?>
```

Comment:
Normally we use `die('skip explanation');`. If there are multiple `if`s the script would continue running.

Reply:
I copy pasted this from one of the other bug tests and didn't double check the section, will amend.